MetabolicAtlas/data-generation
# feat: automatically create gem repository folder

**Related issue(s) and PR(s)**  
This PR closes #28 .

**List of changes made**  
Automatically create a folder for `gemRepository` if it doesn't already exist. This is similar to how we handle the other data output folders.

**Testing**
1. Delete the `gemRepository` folder.
2. Run the data generation, for example: `yarn start ../data-files`.
3. Verify that the `gemRepository` folder is created and the `integratedModelsTimeline.svg` file is in there.
